pages/_app.tsx

```diff
@@ -27,25 +27,32 @@ import '../i18n';
 
 import Layout from 'sections/shared/Layout';
 import { createQueryContext, SynthetixQueryContextProvider } from '@synthetixio/queries';
-import { networkState } from 'store/wallet';
+import Connector from 'containers/Connector';
 
 const InnerApp: FC<AppProps> = ({ Component, pageProps }) => {
-	const network = useRecoilValue(networkState);
+	const { provider, network } = Connector.useContainer();
 
 	return (
 		<>
-			<WithAppContainers>
-				<MediaContextProvider>
-					<SynthetixQueryContextProvider value={createQueryContext({ networkId: network!.id })}>
-						<Layout>
-							<SystemStatus>
-								<Component {...pageProps} />
-							</SystemStatus>
-						</Layout>
-						<ReactQueryDevtools />
-					</SynthetixQueryContextProvider>
-				</MediaContextProvider>
-			</WithAppContainers>
+			<MediaContextProvider>
+				<SynthetixQueryContextProvider
+					value={
+						provider && network
+							? createQueryContext({
+									provider: provider || undefined,
```

Comment:
code scanning says provider always is true here, probably because provider is in the ternary condition?

Reply:
yep sorry I was in a hurry and this became redundant